test/src/tests-store.ts

```diff
@@ -1541,6 +1541,56 @@ describe('Store', () => {
 
                 expect(result).to.deep.equal(expectedResult);
             });
+
+            it('should append url paths to domain when config.domain is set', () => {
```

Comment:
Can you add a test that will handle cnames that don't conform to these new routes, to verify they'll still work?

Reply:
They will always conform to these routes (if what you mean is the portion after `.com/this/is/the/route/all/cname/customers/will/use`